server/events/import_command_runner.go

```diff
@@ -48,6 +51,10 @@ func (v *ImportCommandRunner) Run(ctx *command.Context, cmd *CommentCommand) {
 		ctx.Log.Warn("Error %s", err)
 	}
 
+	if len(projectCmds) == 0 && v.SilenceNoProjects {
```

Comment:
Can this block be tested in `import_command_runner_test.go` ?

Reply:
I added a test case